com.avaloq.tools.ddk.xtext/src/com/avaloq/tools/ddk/xtext/resource/extensions/ResourceDescriptionsUtil.java

```diff
@@ -85,23 +86,25 @@ private ResourceDescriptionsUtil() {}
       }
     }
 
-    return Iterables.filter(descriptions.getAllResourceDescriptions(), input -> {
-      if (matchNames) {
-        for (QualifiedName name : input.getImportedNames()) {
-          if (exportedNames.contains(name.toLowerCase())) { // NOPMD
-            return true;
+    return Iterables.filter(//
+        Iterables.filter(descriptions.getAllResourceDescriptions(), input -> !targetResources.contains(input)) //

```

Comment:
besides formatting, the main change in this class is to add the condition 
input -> !targetResources.contains(input)